File: packages/android/src/main/kotlin/gov/civui/components/CivColumnVisibility.kt
// CivUI — CivColumnVisibility for Jetpack Compose
// Popover for toggling data-grid column visibility. Mirrors
// civ-column-visibility.
//
// Placeholder body — see .claude/rules/audit-debt.md → "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Box
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivColumnVisibility(
    label: String = "",
    minVisible: Int = 1,
    align: String = "end",
    open: Boolean = false,
    modifier: Modifier = Modifier,
    content: @Composable () -> Unit = {},
) {
    Box(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
        content()
    }
}

File: packages/ios/Sources/CivUI/CivColumnVisibility.swift
// CivUI — CivColumnVisibility for SwiftUI
// Popover for toggling data-grid column visibility. Mirrors
// civ-column-visibility.
//
// Placeholder body — the prop surface satisfies schema parity. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivColumnVisibility: View {
    /// Accessible label for the trigger button.
    public var label: String

    /// Minimum number of columns that must stay visible.
    public var minVisible: Int

    /// Popover alignment. `start` | `end`.
    public var align: String

    /// Whether the popover is currently open.
    public var open: Bool

    public init(
        label: String = "",
        minVisible: Int = 1,
        align: String = "end",
        open: Bool = false
    ) {
        self.label = label
        self.minVisible = minVisible
        self.align = align
        self.open = open
    }

    public var body: some View {
        EmptyView()
    }
}
